packages/app/src/state/clients/graphqlClient/subscriptionClient.ts

```diff
@@ -62,13 +69,22 @@ const subscriptionClient = () => {
     }
   });
 
-  return new ApolloClient({
+  subscriptionClient = new ApolloClient({
     cache: new InMemoryCache({
       resultCaching: false
     }),
     link: from([subscriptionErrorLink, wsLink])
   });
+
+  return subscriptionClient;
 };
 
-export default subscriptionClient();
+export const stopSubscriptionClient = () => {
+  subscriptionClient = null;
```

Comment:
shouldn't we technically call `.stop()` if there is a subscriptionClient? This doesn't actually stop it